cql3/functions/functions.cc

```diff
@@ -511,7 +511,8 @@ expr::expression function_call::to_expression() {
 
     return expr::function_call {
         .func = _fun,
-        .args = std::move(args)
+        .args = std::move(args),
```

Comment:
Tip: always terminate lists with a trailing comma, as a courtesy to the next person to append to the list.

Reply:
:+1:, will do in the future